Tests/PackageLoadingTests/ToolsVersionLoaderTests.swift

```diff
@@ -166,7 +166,7 @@ class ToolsVersionLoaderTests: XCTestCase {
                 
                 XCTAssertEqual(
                     error.description,
-                    "the manifest is missing a Swift tools version specification; consider prepending to the manifest '// swift-tools-version:\(ToolsVersion.currentToolsVersion)' to specify the current Swift toolchain version as the lowest Swift version supported by the project; if such a specification already exists, consider moving it to the top of the manifest, or prepending it with '//' to help Swift Package Manager find it"
+					"the manifest is missing a Swift tools version specification; consider prepending to the manifest '// swift-tools-version:\(ToolsVersion.currentToolsVersion < .v5_4 ? "" : " ")\(ToolsVersion.currentToolsVersion.major).\(ToolsVersion.currentToolsVersion.minor)\(ToolsVersion.currentToolsVersion.patch == 0 ? "" : ".\(ToolsVersion.currentToolsVersion.patch)")' to specify the current Swift toolchain version as the lowest Swift version supported by the project; if such a specification already exists, consider moving it to the top of the manifest, or prepending it with '//' to help Swift Package Manager find it"
```

Comment:
alignment intentional?

Reply:
It's unintentional. Space got changed to tabs when I edited the line. Thanks for catching it!